Change the test for LINK_PRIVATE to cmake version 2.8.7.
Cmake 2.8 prior to 2.8.7 does not have LINK_PRIVATE.


diff -Naurb dyninst.orig/common/CMakeLists.txt dyninst/common/CMakeLists.txt
--- dyninst.orig/common/CMakeLists.txt	2014-03-12 18:20:10.000000000 -0500
+++ dyninst/common/CMakeLists.txt	2014-05-28 15:52:00.532800410 -0500
@@ -108,7 +108,7 @@
 set_target_properties (common common_static PROPERTIES PUBLIC_HEADER "${headers}")
 
 if (${IBERTY_LIBRARY} MATCHES iberty)
-if(${CMAKE_VERSION} VERSION_LESS "2.8")
+if(${CMAKE_VERSION} VERSION_LESS "2.8.7")
 target_link_libraries (common ${IBERTY_LIBRARY})
 else()
 target_link_libraries (common LINK_PRIVATE ${IBERTY_LIBRARY})
